### PR TITLE
replace statsmodels with scipy stats

### PR DESCRIPTION
**Pull request by martinkim0**

### CODE REVIEW

File: src/scvi/external/mrvi/_model.py [hunk 1]
```diff
@@ -1112,7 +1112,7 @@ def differential_expression(
         """
         from functools import partial
 
-        from statsmodels.stats.multitest import multipletests
+        from scipy.stats import false_discovery_rate
```
Comment (justjhong) [inline src/scvi/external/mrvi/_model.py:1115]: should be false_discovery_control, not rate. let's make sure this one passes tests.

**Comment by martinkim0** [comment]
Tests are currently failing because `pvalue` has NaNs (before `false_discovery_control` is called). This wasn't being caught before for some reason (does statsmodels not check for NaNs?)

**Comment by justjhong** [comment]
@martinkim0 good catch, looks like statsmodels would just output nans. This happens when the input to chi2.cdf is 0 (which usually shouldn't happen -  happens here since its barely training I assume), but we can still correctly handle this case by making the pval 1 for these examples.

**Review by justjhong — APPROVED** [review]

**Comment by martinkim0** [comment]
Awesome, thanks for pushing the changes

**Pull request merged**